### PR TITLE
[WB-4251] extend timeout for fault injection test

### PR DESCRIPTION
**Pull request by raubitsj**
https://wandb.atlassian.net/browse/WB-4251
(*NOTE*: this only fixes one of the flaky tests mentioned in the above ticket.)

Description
-----------

Test (windows py37 wandb_integration_test.py test_network_fault_graphql) is creating multiple graphql failures (5) and the timeout/retry logic on windows seems to be causing this test to take over 60 seconds to recover and retry network connections.  It should behave the same on other platforms but it does not.

Test extends the timeout on all platforms to 120 seconds (default is 60 seconds)

Excerpt from a failing run, notice the network takes 40 seconds to recover which leaves a narrow window to finish:
```
wandb: Tracking run with wandb version 0.10.15.dev1
wandb: Syncing run lovely-dawn-32
wandb:  View project at http://localhost/mock_server_entity/test
wandb:  View run at http://localhost/mock_server_entity/test/runs/27h89pt9
wandb: Run data is saved locally in C:\Users\circleci\project\tests\logs\test_network_fault_graphql\wandb\run-20210118_205831-27h89pt9
wandb: Run `wandb offline` to turn off syncing.
wandb: Waiting for W&B process to finish, PID 5204
wandb: Program ended successfully.
wandb: ERROR Error while calling W&B API: Server down (<Response [500]>)
wandb: ERROR Error while calling W&B API: Server down (<Response [500]>)
wandb: ERROR Error while calling W&B API: Server down (<Response [500]>)
wandb: Network error (HTTPError), entering retry loop. See wandb\debug-internal.log for full traceback.
wandb: ERROR Error while calling W&B API: Server down (<Response [500]>)
wandb: ERROR Error while calling W&B API: Server down (<Response [500]>)
wandb: Network error resolved after 0:00:40.359711, resuming normal operation.
wandb:                                                                                
```

Testing
-------

How was this PR tested?

Tested by looping the windows circle testing:
```
./tools/circleci-tool.py --platform win --wait --loop 3
```

For some reason the test seems to fail more often if the entire test suite is run, not if a single file or single test is run in a loop

### CODE REVIEW

File: tests/wandb_integration_test.py [hunk 1]
```diff
@@ -183,6 +183,7 @@ def test_network_fault_files(live_mock_server, test_settings):
     )
 
 
+@pytest.mark.timeout(120)
 def test_network_fault_graphql(live_mock_server, test_settings):
```
Comment (vanpelt) [inline tests/wandb_integration_test.py:187]: Hmmm, I wonder why it's taking that long.  Is it the backoff?  We could probably decrease `fail_graphql_times`...

Comment (raubitsj) [reply tests/wandb_integration_test.py:187]: Yeah, i just want to get these tests less flaky.  I don't want to defeature functionality as I am not sure what the 5 graphql requests are right now, (viewer query - maybe *2, then an upsert? * 3?)

We can take another pass making the unittests more efficiently use time.

Comment (raubitsj) [reply tests/wandb_integration_test.py:188]: I added a ticket and a comment to look into this in the future:
https://wandb.atlassian.net/browse/WB-4365

**Review by KyleGoyette — APPROVED** [review]
All right, looks good to me. We can look into fail_graphql_times in the future.

**Pull request merged**